google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkCommitBatch.java

```diff
@@ -34,159 +34,108 @@
 import java.util.List;
 import java.util.Set;
 import java.util.concurrent.CopyOnWriteArraySet;
-import javax.annotation.Nullable;
+import java.util.concurrent.Executor;
 
 /** Used to represent a batch on the BatchQueue. */
```

Comment:
Update class name?
My attempt: `A batch that holds and schedules BulkWriter operations.`

Reply:
The main issue I had was that "BatchQueue" is no longer a term used in BulkWriter.